### PR TITLE
Cleanup codegen subproject scripts

### PR DESCRIPTION
**Pull request by thockin**
Mostly really small things like linewraps and quoting.

/kind cleanup
/priority backlog
/sig architecture

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh [hunk 1]
```diff
@@ -25,14 +25,17 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-
...
   "wardle:v1alpha1,v1beta1" \
   --output-base "$(dirname "${BASH_SOURCE[0]}")/../../.." \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate.go.txt
 
-bash "${CODEGEN_PKG}/generate-internal-groups.sh" "deepcopy,defaulter,conversion,openapi" \
-  k8s.io/sample-apiserver/pkg/generated k8s.io/sample-apiserver/pkg/apis k8s.io/sample-apiserver/pkg/apis \
+"${CODEGEN_PKG}/generate-internal-groups.sh" "deepcopy,defaulter,conversion,openapi" \
+  k8s.io/sample-apiserver/pkg/generated \
+  k8s.io/sample-apiserver/pkg/apis \
+  k8s.io/sample-apiserver/pkg/apis \
```
Comment (aojea) [inline staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh:38]: why the same path twice?

Comment (aojea) [reply staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh:38]: I see 
```
Usage: $(basename $0) <generators> <output-package> <internal-apis-package> <extensiona-apis-package> <groups-versions> ...
```

Comment (thockin) [reply staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh:38]: Yeah, I hate these scripts, but ... shell...

**Pull request merged**